docs/_documentations/troubleshooting.md

```diff
@@ -390,6 +390,33 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Codewind version: 0.6.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1061
+-->
+## Appsody binds fail
+
+If an Appsody respository is already added in your local Appsody CLI, and if you try to add the Appsody repository again with a different name or ID, the Appsody bind might fail in the following scenario:
+1. Use a template manager to add the Appsody repository in Codewind.
+2. Try to bind the project to a stack from the added Appsody repository.
+3. Create the project in Codewind.
```

Comment:
I guess this should say something about the project will appear in Codewind explorer rather than "Create", since this is binding a project that already exists on the disk